qutebrowser/app.py

```diff
@@ -333,6 +333,13 @@ def open_url(url, target=None, no_raise=False, via_ipc=True):
     tabbed_browser = objreg.get('tabbed-browser', scope='window',
                                 window=win_id)
     log.init.debug("About to open URL: {}".format(url.toDisplayString()))
+
+    if config.val.tabs.switch_to_open_url:
+        cd = commands.CommandDispatcher(win_id, tabbed_browser)
```

Comment:
It seems quite odd that `app.py` needs to create a `CommandDispatcher` for something which isn't actually a command being run (also, I'd like to get rid of `commands.py` entirely at some point: #1129).

Also see my comments below.

Reply:
Moved.